### PR TITLE
Feature mute stream log for info level

### PR DESCRIPTION
**Pull request by azurewtl**
**Features**
see issue : #1109 
llm stream_response alway been print regardless of debug flag
I modified `logs.py` rather than `config2.py`.
This is because log is been set via `define_log_level` rather than `config.yaml`
To keep things simple, a global variable `_print_level` is introduced in `logs.py`
    
**Influence**
`define_log_level(print_level="INFO")` will mute `log_llm_stream`

### CODE REVIEW

File: metagpt/logs.py [hunk 1]
```diff
@@ -31,7 +35,8 @@ def define_log_level(print_level="INFO", logfile_level="DEBUG", name: str = None
 
 
 def log_llm_stream(msg):
-    _llm_stream_log(msg)
+  if _print_level in ["DEBUG"]:
+        _llm_stream_log(msg)
```
Comment (shenchucheng) [inline metagpt/logs.py:39]: It's recommended to add this condition inside the `_llm_stream_log` function. Users can utilize `set_llm_stream_logfunc` to customize the behavior of their own `llm_stream_log` , but they may not want it to be affected by the `log_level` flag. You can define the function like this:

```python

def _llm_stream_log(msg):
    if _print_level in ["DEBUG"]:
        print(msg, end="")
```

**Review by shenchucheng — CHANGES_REQUESTED** [review]

File: metagpt/logs.py (hunk 1 above)
Comment (azurewtl) [reply metagpt/logs.py]: reasonable consideration！ change is made accordingly

**Comment by shenchucheng** [comment]
@azurewtl Thanks for your improvements. It works when I call `define_log_level(print_level="DEBUG")` in my Python script entry if I want to print the LLM stream log. However, I'm considering whether to provide a simpler way to set it up, perhaps by passing environment variables `METAGPT_LOG_PRINT_LEVEL` and `METAGPT_LOG_FILE_LEVEL`?

**Comment by azurewtl** [comment]
> @azurewtl Thanks for your improvements. It works when I call `define_log_level(print_level="DEBUG")` in my Python script entry if I want to print the LLM stream log. However, I'm considering whether to provide a simpler way to set it up, perhaps by passing environment variables `METAGPT_LOG_PRINT_LEVEL` and `METAGPT_LOG_FILE_LEVEL`?

I thought about using config. however, my proposal is that debug flag should be pass via command line tools.
Oftentimes, If we want to see debug info during development, I run program and pass debug flag such as:
```
python run.py --debug
``` 
Maybe YAML better off storage config that is NOT changed on ad-hoc basis. Otherwise if I want to show less or more info, I need to change config frequently.

**Comment by geekan** [comment]
@shenchucheng I think this is a new requirement and can be discussed additionally. This PR can be merged first

**Review by geekan — APPROVED** [review]

File: metagpt/logs.py [hunk 2]
```diff
@@ -14,9 +14,13 @@
 
 from metagpt.const import METAGPT_ROOT
 
+_print_level = "INFO"
```
Comment (geekan) [inline metagpt/logs.py:17]: 默认的 _print_level 应该使用 DEBUG？

**Comment by geekan** [comment]
Anyway, I’ll merge first

**Pull request merged**